core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -515,10 +523,14 @@ trait EntityComponent {
     def copyEntities(sourceWorkspaceContext: SlickWorkspaceContext, destWorkspaceContext: SlickWorkspaceContext, entityType: String, entityNames: Seq[String]): ReadWriteAction[TraversableOnce[Entity]] = {
       getEntitySubtrees(sourceWorkspaceContext, entityType, entityNames).flatMap { entities =>
         getCopyConflicts(destWorkspaceContext, entities).flatMap { conflicts =>
-          if (conflicts.isEmpty) {
-            cloneEntities(destWorkspaceContext, entities).map(_ => Seq.empty[Entity])
+          // trueConflicts are the entities that conflict based on user-specified input. These are fatal.
+          // subtreeConflicts are the entities that conflict after we look up the subtrees. These are non-fatal, because we can simply link them up with the user-specified entities.
+          val trueConflicts = conflicts.filter(e => (e.entityType.equalsIgnoreCase(entityType) && entityNames.contains(e.name))).toSeq
+          val subtreeConflicts = conflicts.toSeq diff trueConflicts
```

Comment:
Imagine this scenario:

WorkspaceA has the entity participant1
WorkspaceB has the entities sample1 and participant1

If sample1 references participant1, and you try to copy sample1 from WorkspaceB into WorkspaceA, you will get a conflict error because the copy operation will try to copy over sample1 and all of the entities that it holds references to. The behavior here is changed to try to copy over all referenced entities, but if they already exist in the destination workspace, they will be linked with the new entities instead of failing.

So copying sample1 from WorkspaceB to WorkspaceA will see that a participant1 already exists, and create a reference between the new sample1 and the already existing participant1.

`subtreeConflicts` are these types conflicts- they're entities that we don't want to fail the operation for but also we don't want to copy them over a 2nd time.